subsys/bluetooth/audio/csip_set_member.c

```diff
@@ -652,10 +631,8 @@ static struct bt_conn_cb conn_callbacks = {
 	.security_changed = csip_security_changed,
 };
 
-static struct bt_conn_auth_info_cb auth_callbacks = {
-	.pairing_complete = auth_pairing_complete,
-	.bond_deleted = csip_bond_deleted
-};
+static struct bt_conn_auth_info_cb auth_callbacks = {.pairing_complete = auth_pairing_complete,
```

Comment:
Is this really done on purpuse? This is not how we usually format a statically defined callback structure!? Pre-changed looks more consistent

Reply:
Welp, my editor done goofed and formatted the entire file. Will revert the formatting lines

Reply:
Haha, I was pondering whether I should make a comment on that - it did seem like a lot of formatting without mentioning it :D